actions/actions.go

```diff
@@ -1,5 +1,35 @@
 package actions
 
+/*
+	 The actions system is a key component for process automation in WIT. It provides
+	 a way of executing user-configurable, dynamic process steps depending on user
+	 settings, schema settings and events in the WIT. 
+
+	 The idea here is to provide a simple, yet powerful "signal-slot" system that
```

Comment:
The wording *signal-slot* is only known to people with a Qt background. Should we call this publish/subscribe system maybe?

Reply:
Done.

Reply:
Done.